target/ppc/int_helper.c

```diff
@@ -910,6 +910,23 @@ VCT(uxs, cvtsduw, u32)
 VCT(sxs, cvtsdsw, s32)
 #undef VCT
 
+/*
```

Comment:
I don't think we should define these functions. Testing directly against the enum options should be simple enough.